### DISCUSSION TITLE
DELETE

### DISCUSSION THREAD

**Discussion by mellis481** [community/community#39379] [Other Feature Feedback, Questions, & Ideas] [answered]

**Comment by laughedelic** [comment, marked as answer]
Aren't these two settings the same? It's just different wording:
- `off`: admins and other roles with the "bypass" permission can skip the protection
- `on`: nobody can skip protection rules:
  - "include administrators": do not allow them to bypass
  - "do not allow bypassing": same thing, include administrators to those who cannot bypass

**Reply by mellis481** [reply]
@laughedelic Doh. I guess this is what happens when I post a message when I'm exhausted.

**Reply by laughedelic** [reply]
No worries! But I think rather than deleting the question, it would be more useful to keep it and mark as solved. At least it would provide some value to others who might get confused by this new wording.

**Reply by Gega09** [reply]
Delete pls

**Comment by ghostinhershell** [comment]
Hey @mellis481 👋 

Hope all is well and I'm happy that you were able to get your question answered. I'd like to echo @laughedelic's response above:
> No worries! But I think rather than deleting the question, it would be more useful to keep it and mark as solved. At least it would provide some value to others who might get confused by this new wording.

Restoring the text from your original question, and marking @laughedelic's response as the answer to this post, will help anyone else in the community with a similar question in the future. Thanks! 🚀
